back-end/src/modules/auth/user/User.ts

```diff
@@ -0,0 +1,72 @@
+import { Required  } from '@tsed/common';
+import { Model, ObjectID, PreHook, Unique } from '@tsed/mongoose';
+import { hash, compare } from 'bcrypt';
+import { isNull } from 'lodash';
+
+export class UserCreateUpdate {
+    @Required()
+    public firstName: string;
+    @Required()
+    public lastName: string;
+    @Required()
+    public email: string;
+    @Required()
+    public password: string;
+}
+
+export class GetUser {
```

Comment:
Where are you using this?

Reply:
helemaal niet. hahaha, ik heb geen functionaliteiten om de user op te vragen though. maar get User wordt verwijdert